### DISCUSSION TITLE
How to find out if a photon update is available

### DISCUSSION THREAD

**Discussion by Tealk** [Xyphyn/photon#469] [Q&A] [answered]
Hi,

because lemmy v0.19.6 has just been released, I wanted to ask how you can find out if photon is ready for a new lemmy version.
I only found one issue: https://github.com/Xyphyn/photon/issues/459

**Comment by Xyphyn** [comment, marked as answer]
Photon and all of its *current* features will work with v0.19.6 for sure but any new features id have to update photon for.

**Reply by Tealk** [reply]
I have noticed that some images are only displayed as thumbnails
